test/serializers/serialize_exercise_instructions_test.rb

```diff
@@ -0,0 +1,96 @@
+require 'test_helper'
+
+class SerializeExerciseInstructionsTest < ActiveSupport::TestCase
+  test "serialize general hints" do
+    track = create :track, slug: 'fsharp'
+    exercise = create :concept_exercise, track: track, slug: 'log-levels'
+
+    serialized = SerializeExerciseInstructions.(exercise)
+
+    expected = [
+      "<p>The <code>string</code> class has many useful <a href=\"https://docs.microsoft.com/en-us/dotnet/api/system.string?view=netcore-3.1#methods\" target=\"_blank\">built-in\nmethods</a>.</p>\n", # rubocop:disable Layout/LineLength
+      "<p>Remember that strings are immutable.</p>\n"
```

Comment:
We currently have a `\n` at the end, which is how the Markdown is rendered. I _could_ strip that off if you'd like that

Reply:
I think it's fine as it is.